### DISCUSSION TITLE
Mica in react native windows

### DISCUSSION THREAD

**Discussion by ItsPi3141** [microsoft/react-native-windows#11175] [Q&A] [answered]
Can't seem to find a way to use mica. Do I need to use winui3 instead of winui2?

I've seen this discussion but it's for acrylic:
https://github.com/microsoft/react-native-windows/discussions/9483

**Comment by chrisglein** [comment]
You should be able to use Mica with WinUI2. But I believe all the built in brushes for WinUI2 use Acrylic. But this should all be possible. Just not made easy through PlatformColor. So the information in the discussion you linked should be a good start.

**Comment by DanielSRS** [comment, marked as answer]
To use Mica you just have to edit the windows/\<your app name\>/MainPage.xaml file like the following:

```diff
<Page
    x:Class="myapp.MainPage"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:myapp"
    xmlns:react="using:Microsoft.ReactNative"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+   xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
    mc:Ignorable="d"
-   Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+   muxc:BackdropMaterial.ApplyToRootOrPageBackground="True">
    <react:ReactRootView 
        x:Name="ReactRootView"
        ComponentName="myapp"
-       Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+       Background="Transparent"
        MinHeight="400"/>
</Page>
```

- **In order to see the Mica, the views in your app must be transparent**
- Tested with a C++ project
- Acrylic still works, so your app window can have Acrylic or Mica
